diff -ru vulture.orig/admin/vulture_httpd.conf vulture-2.0.3/admin/vulture_httpd.conf
--- vulture.orig/admin/vulture_httpd.conf	2012-04-16 18:17:46.000000000 -0400
+++ vulture-2.0.3/admin/vulture_httpd.conf	2012-04-16 18:12:54.000000000 -0400
@@ -21,7 +21,7 @@
 user {{ www_user }}
 group {{ www_user }}
 
-LoadFile                       /usr/lib/libxml2.so.2
+LoadFile                       /usr/lib64/libxml2.so.2
 
 LoadModule apreq_module {{ serverroot }}/modules/mod_apreq2.so
 LoadModule perl_module		{{ serverroot }}/modules/mod_perl.so
