diff -ru '--exclude=.svn' vulture//lib/Vulture/Core/FixupHandler.pm vulture-suse//lib/Vulture/Core/FixupHandler.pm
--- vulture//lib/Vulture/Core/FixupHandler.pm	2012-12-31 16:38:06.000000000 +0100
+++ vulture-suse//lib/Vulture/Core/FixupHandler.pm	2013-01-02 15:31:44.000000000 +0100
@@ -12,9 +12,10 @@
 use Apache2::Log;
 
 use Apache2::Const -compile => qw(OK);
-use Apache::SSLLookup;
+use Core::VultureUtils qw(&load_module); 
 
 sub handler {
+    Core::VultureUtils::load_module('Apache::SSLLookup','plugin');
     my $r   = Apache::SSLLookup->new(shift);
     my $log = $r->pnotes('log');
 
diff -ru '--exclude=.svn' vulture//lib/Vulture/Core/ResponseHandlerv2.pm vulture-suse//lib/Vulture/Core/ResponseHandlerv2.pm
--- vulture//lib/Vulture/Core/ResponseHandlerv2.pm	2012-12-31 16:38:06.000000000 +0100
+++ vulture-suse//lib/Vulture/Core/ResponseHandlerv2.pm	2013-01-02 15:21:03.000000000 +0100
@@ -19,10 +19,10 @@
   qw(&session &get_style &get_translations &generate_random_string &load_module);
 use SSO::ProfileManager qw(&get_profile);
 
-use Apache::SSLLookup;
 use HTML::Entities qw(&encode_entities);
 
 sub handler {
+    Core::VultureUtils::load_module('Apache::SSLLookup','plugin');
     my $r = Apache::SSLLookup->new(shift);
 
     my $log = $r->pnotes('log');
diff -ru '--exclude=.svn' vulture//lib/Vulture/Core/TransHandlerv2.pm vulture-suse//lib/Vulture/Core/TransHandlerv2.pm
--- vulture//lib/Vulture/Core/TransHandlerv2.pm	2012-12-31 16:38:06.000000000 +0100
+++ vulture-suse//lib/Vulture/Core/TransHandlerv2.pm	2013-01-02 15:32:39.000000000 +0100
@@ -11,7 +11,6 @@
 use Apache2::RequestRec;
 use Apache2::RequestIO;
 use Apache2::RequestUtil;
-use Apache::SSLLookup;
 use Apache2::Const -compile => qw(OK FORBIDDEN REDIRECT DONE NOT_FOUND);
 use Core::VultureUtils qw(&get_memcached_conf &get_app &get_intf &version_check &get_cookie 
    &session &get_translations &get_style &load_module 
@@ -28,6 +27,7 @@
 
 sub handler {
 
+    Core::VultureUtils::load_module('Apache::SSLLookup','plugin');
     my $r            = Apache::SSLLookup->new(shift);
     my $uri          = $r->uri;
     my $unparsed_uri = $r->unparsed_uri;
diff -ru '--exclude=.svn' vulture//rpm/httpd.conf vulture-suse//rpm/httpd.conf
--- vulture//rpm/httpd.conf	2012-11-22 11:43:49.000000000 +0100
+++ vulture-suse//rpm/httpd.conf	2013-01-02 15:22:59.000000000 +0100
@@ -1,4 +1,4 @@
-ServerRoot /etc/httpd
+ServerRoot /etc/apache2
 
 Listen 127.0.0.1:9090
 LoadModule      wsgi_module             modules/mod_wsgi.so
@@ -12,8 +12,8 @@
 
 WSGIScriptAlias / /opt/vulture/conf/vulture.wsgi
 
-user apache
-group apache
+user wwwrun
+group www
 
 PidFile /opt/vulture/conf/vulture.pid
 
diff -ru '--exclude=.svn' vulture//rpm/settings.py vulture-suse//rpm/settings.py
--- vulture//rpm/settings.py	2013-01-02 15:28:42.000000000 +0100
+++ vulture-suse//rpm/settings.py	2013-01-02 15:24:40.000000000 +0100
@@ -22,16 +22,16 @@
    '/opt/vulture/admin/fixtures/',
 )
 
-HTTPD_PATH = 'sudo /usr/sbin/httpd'
+HTTPD_PATH = 'sudo /usr/sbin/httpd2'
 BIN_PATH = '/opt/vulture/bin/'
 
 CONF_PATH = '/opt/vulture/conf/'
 DATABASE_PATH = '/opt/vulture/admin/'
-SERVERROOT = '/etc/httpd'
+SERVERROOT = '/etc/apache2'
 HTTPD_CUSTOM = ""
 
-WWW_USER='apache'
-WWW_GROUP='apache'
+WWW_USER='wwwrun'
+WWW_GROUP='www'
 
 PERL_SWITCHES = '-I/opt/vulture/lib/i386-linux-thread-multi -I/opt/vulture/lib/i386-linux-thread-multi/Vulture -I/opt/vulture/lib/x86_64-linux-thread-multi -I/opt/vulture/lib/i486-linux-thread-multi -I/opt/vulture/lib/i586-linux-thread-multi -I/opt/vulture/lib/i486-linux-gnu-thread-multi -I/opt/vulture/lib -I/opt/vulture/usr/local/lib64/perl5 -I/opt/vulture/usr/lib/perl5 -I/opt/vulture/usr/lib64/perl5 -I/opt/vulture/usr/lib64/perl5/site_perl/5.8.8/x86_64-linux-thread-multi/Apache2'
 
