--- vulture-orig/bin/update-rules.sh	2012-04-16 16:34:19.000000000 -0400
+++ vulture-2.0.3/bin/update-rules.sh	2012-04-17 09:08:12.000000000 -0400
@@ -8,7 +8,7 @@
 #######################################
 #######################################
 
-BASEDIR="/var/www/vulture"
+BASEDIR="/opt/vulture"
 CONFDIR=$BASEDIR/conf
 FETCH_SCRIPT=$BASEDIR/bin/rules-updater.pl

--- vulture-orig/admin/vulture_httpd.conf	2012-05-23 06:46:12.000000000 -0400
+++ vulture-2.0.3/admin/vulture_httpd.conf	2012-05-23 06:29:10.000000000 -0400
@@ -23,16 +23,23 @@
 user {{ www_user }}
 group {{ www_user }}
 
-LoadFile			/usr/lib/libxml2.so.2
+LoadFile			/usr/lib64/libxml2.so.2
 LoadModule perl_module		{{ serverroot }}/modules/mod_perl.so
 LoadModule apreq_module    {{ serverroot }}/modules/mod_apreq2.so
 LoadModule env_module           {{ serverroot }}/modules/mod_env.so
+LoadModule setenvif_module      {{ serverroot }}/modules/mod_setenvif.so
+LoadModule alias_module         {{ serverroot }}/modules/mod_alias.so
+LoadModule log_config_module    {{ serverroot }}/modules/mod_log_config.so
+LoadModule ssl_module           {{ serverroot }}/modules/mod_ssl.so
+LoadModule authz_host_module    {{ serverroot }}/modules/mod_authz_host.so
+LoadModule proxy_module         {{ serverroot }}/modules/mod_proxy.so
+LoadModule proxy_ftp_module     {{ serverroot }}/modules/mod_proxy_ftp.so
+LoadModule proxy_http_module    {{ serverroot }}/modules/mod_proxy_http.so
+LoadModule proxy_connect_module {{ serverroot }}/modules/mod_proxy_connect.so
 {{ httpd_custom|safe }}
 LoadModule proxy_balancer_module	{{ serverroot }}/modules/mod_proxy_balancer.so
 ## Mod security rules
-{% if app.security.all %}
-    LoadModule security2_module     {{ serverroot }}/mod_security2.so\n\
-{% endif %}
+LoadModule security2_module     {{ serverroot }}/modules/mod_security2.so
 LoadModule headers_module	{{serverroot}}/modules/mod_headers.so
 LoadModule unique_id_module	{{serverroot}}/modules/mod_unique_id.so
 SecDataDir {{ VultureConfPath }}security-rules/
