diff -ru Apache-SSLLookup-2.00_04.old/Makefile.PL Apache-SSLLookup-2.00_04/Makefile.PL
--- Apache-SSLLookup-2.00_04.old/Makefile.PL	2005-04-06 16:10:13.000000000 +0200
+++ Apache-SSLLookup-2.00_04/Makefile.PL	2005-11-02 13:32:16.570711952 +0100
@@ -1,6 +1,6 @@
 #!perl
 
-use 5.008001;
+use 5.008000;
 
 use ModPerl::MM ();
 use Apache::TestMM qw(test clean);
diff -ru Apache-SSLLookup-2.00_04.old/SSLLookup.pm Apache-SSLLookup-2.00_04/SSLLookup.pm
--- Apache-SSLLookup-2.00_04.old/SSLLookup.pm	2005-06-16 15:30:58.000000000 +0200
+++ Apache-SSLLookup-2.00_04/SSLLookup.pm	2005-11-02 13:33:05.936207256 +0100
@@ -1,6 +1,6 @@
 package Apache::SSLLookup;
 
-use 5.008001;
+use 5.008000;
 
 use strict;
 
