diff -ruN Apache2-AuthenNTLM-0.02.ori/smb/smbval/smblib-util.c Apache2-AuthenNTLM-0.02/smb/smbval/smblib-util.c
--- Apache2-AuthenNTLM-0.02.ori/smb/smbval/smblib-util.c	2005-04-28 21:14:03.000000000 +0200
+++ Apache2-AuthenNTLM-0.02/smb/smbval/smblib-util.c	2011-10-31 14:40:11.000000000 +0100
@@ -29,6 +29,10 @@
 #else
 #include <malloc.h>
 #endif
+#include <string.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <sys/socket.h>
 
 #include "rfcnb.h"
 
@@ -793,5 +797,5 @@
 
     }
   }
-
+return 0;
 }
diff -ruN Apache2-AuthenNTLM-0.02.ori/smb/smbval/valid.c Apache2-AuthenNTLM-0.02/smb/smbval/valid.c
--- Apache2-AuthenNTLM-0.02.ori/smb/smbval/valid.c	2005-04-28 21:14:03.000000000 +0200
+++ Apache2-AuthenNTLM-0.02/smb/smbval/valid.c	2011-10-31 13:50:49.000000000 +0100
@@ -3,6 +3,10 @@
 #include <syslog.h>
 #include "smblib-priv.h"
 #include "valid.h"
+#include <string.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <sys/socket.h>
 
 SMB_Handle_Type SMB_Connect_Server(void *, char *, char *);
 
