diff -ruN vulture-2.0.1.ori/Makefile vulture-2.0.1/Makefile
--- vulture-2.0.1.ori/Makefile	2011-10-31 18:06:44.000000000 +0100
+++ vulture-2.0.1/Makefile	2011-11-12 21:54:53.000000000 +0100
@@ -107,16 +107,28 @@
 	for i in '$(DIRS)'; do \
 		for j in `find $$i`; do \
 		if [ -f $$j ]; then \
-			$(INSTALL) $(UID) $(GID) $$j $(DESTDIR)$(PREFIX)/$(NAME)/$$j; \
+			if [ -e /etc/debian_version ]; then \
+				$(INSTALL) $(UID) $(GID) $$j $(DESTDIR)$(PREFIX)/$(NAME)/$$j; \
+			else \
+				$(INSTALL) $$j $(DESTDIR)$(PREFIX)/$(NAME)/$$j; \
+			fi; \
 		fi; \
 		done; \
 	done
-	/usr/bin/install -d -m0755 $(UID) $(GID) $(DESTDIR)$(PREFIX)/$(NAME)/sql
+	if [ -e /etc/debian_version ]; then \
+		/usr/bin/install -d -m0755 $(UID) $(GID) $(DESTDIR)$(PREFIX)/$(NAME)/sql; \
+	else \
+		/usr/bin/install -d -m0755 $(DESTDIR)$(PREFIX)/$(NAME)/sql; \
+	fi
 	for i in '$(DIRSLIB)'; do \
 		cd lib/Vulture; \
 		for j in `find $$i`; do \
 		if [ -f $$j ]; then \
-			$(INSTALL) $(UID) $(GID) $$j $(DESTDIR)$(PREFIXLIB)/$(NAME)/lib/i386-linux-thread-multi/Vulture/$$j; \
+			if [ -e /etc/debian_version ]; then \
+				$(INSTALL) $(UID) $(GID) $$j $(DESTDIR)$(PREFIXLIB)/$(NAME)/lib/i386-linux-thread-multi/Vulture/$$j; \
+			else \
+				$(INSTALL) $$j $(DESTDIR)$(PREFIXLIB)/$(NAME)/lib/i386-linux-thread-multi/Vulture/$$j; \
+			fi; \
 		fi; \
 		done; \
 		cd ../..; \
