--- vulture-2.0.4.ori/admin/vulture_httpd.conf	2012-07-18 16:05:15.861716420 -0400
+++ vulture-2.0.4/admin/vulture_httpd.conf	2012-07-18 16:06:30.384715654 -0400
@@ -23,10 +23,20 @@
 user {{ www_user }}
 group {{ www_user }}
 
-LoadFile                        /usr/lib/libxml2.so.2
+LoadFile			/usr/lib64/libxml2.so.2
 LoadModule perl_module          {{ serverroot }}/modules/mod_perl.so
 LoadModule apreq_module    {{ serverroot }}/modules/mod_apreq2.so
 LoadModule env_module           {{ serverroot }}/modules/mod_env.so
+LoadModule setenvif_module      {{ serverroot }}/modules/mod_setenvif.so
+LoadModule alias_module         {{ serverroot }}/modules/mod_alias.so
+LoadModule log_config_module    {{ serverroot }}/modules/mod_log_config.so
+LoadModule ssl_module           {{ serverroot }}/modules/mod_ssl.so
+LoadModule authz_host_module    {{ serverroot }}/modules/mod_authz_host.so
+LoadModule proxy_module         {{ serverroot }}/modules/mod_proxy.so
+LoadModule proxy_ftp_module     {{ serverroot }}/modules/mod_proxy_ftp.so
+LoadModule proxy_http_module    {{ serverroot }}/modules/mod_proxy_http.so
+LoadModule proxy_connect_module {{ serverroot }}/modules/mod_proxy_connect.so
+ 
 {{ httpd_custom|safe }}
 LoadModule proxy_balancer_module        {{ serverroot }}/modules/mod_proxy_balancer.so
 {% if MS_path.value %}
