--- freetds-0.82/include/cspublic.h.orig	2010-01-20 19:48:21.000000000 +0100
+++ freetds-0.82/include/cspublic.h	2010-01-20 19:48:30.000000000 +0100
@@ -34,7 +34,7 @@
 #define TDS_STATIC_CAST(type, a) ((type)(a))
 #endif
 
-static const char rcsid_cspublic_h[] = "$Id: cspublic.h,v 1.58 2007/06/25 09:48:20 freddy77 Exp $";
+static const char rcsid_cspublic_h[] = "$Id: cspublic.h,v 1.61 2008/09/08 17:50:25 jklowden Exp $";
 static const void *const no_unused_cspublic_h_warn[] = { rcsid_cspublic_h, no_unused_cspublic_h_warn };
 
 #define CS_PUBLIC
@@ -446,7 +446,10 @@
 #define CS_VERSION_150	15000
 
 #define BLK_VERSION_100 CS_VERSION_100
-#define BLK_VERSION_110 CS_VERSION_100
+#define BLK_VERSION_110 CS_VERSION_110
+#define BLK_VERSION_120 CS_VERSION_120
+#define BLK_VERSION_125 CS_VERSION_125
+#define BLK_VERSION_150 CS_VERSION_150
 
 #define CS_FORCE_EXIT	300
 #define CS_FORCE_CLOSE  301
@@ -746,6 +749,8 @@
 CS_RETCODE cs_time(CS_CONTEXT * ctx, CS_LOCALE * locale, CS_VOID * buffer, CS_INT buflen, CS_INT * outlen, CS_DATEREC * daterec);
 CS_RETCODE cs_will_convert(CS_CONTEXT * ctx, CS_INT srctype, CS_INT desttype, CS_BOOL * result);
 
+const char * cs_prretcode(int retcode);
+
 #ifdef __cplusplus
 #if 0
 {
