diff -ruN vulture-2.0.2.ori/admin/vulture_httpd.conf vulture-2.0.2/admin/vulture_httpd.conf
--- vulture-2.0.2.ori/admin/vulture_httpd.conf	2011-12-07 19:02:26.000000000 +0100
+++ vulture-2.0.2/admin/vulture_httpd.conf	2011-12-07 20:35:54.000000000 +0100
@@ -24,7 +24,7 @@
 user {{ www_user }}
 group {{ www_user }}
 
-LoadFile			/usr/lib/libxml2.so.2
+LoadFile			/usr/lib64/libxml2.so.2
 LoadModule perl_module		{{ serverroot }}/modules/mod_perl.so
 {{ httpd_custom|safe }}
 ## Mod security rules
